yarn-project/stdlib/src/tx/private_execution_result.ts

```diff
@@ -251,10 +249,10 @@ export function collectSortedContractClassLogs(execResult: PrivateExecutionResul
  * @param execResult - The execution result to collect messages from.
```

Comment:
```suggestion
 * @param execResult - The execution result to collect offchain effects from.
```

Also the first sentence needs to be updated. Github UI doesn't allow me to modify it as it's too far from the change.